### ISSUE TITLE
Improve documentation for LayersControl

### ISSUE THREAD

**Issue by undergroundquizscene** [jupyter-widgets/ipyleaflet#293]
In [the LayersControl documentation](https://ipyleaflet.readthedocs.io/en/latest/api_reference/layers_control.html), it seems to suggest that you add a new `LayersControl` object, initialised with no parameters, and you automatically get control over the layers.

However, when I do this, I only get one checkbox in the control, which is unlabelled and only controls the most-recently added `LayerGroup`. There doesn’t seem to be any way to control older groups, or give names in the label.

Is there something I’ve missed?

**Comment by martinRenou** [comment]
Would you mind sharing an example of code that reproduces the issue?

**Comment by undergroundquizscene** [comment]
Yeah, no problem – I’ll get an example together and post it later today :)

**Comment by undergroundquizscene** [comment]
Sorry for the delay! Here is code that reproduces the problem for me:

```python
# cell 1
import ipyleaflet as lf
themap = lf.Map(center=(51.89217, -8.55789), zoom=13)
themap
```

```python
# cell 2
stops = [(51.88801, -8.59634),
 (51.88807, -8.59077),
 (51.88872, -8.58343),
 (51.88996, -8.57499),
 (51.8923, -8.56321),
 (51.89217, -8.55789),
 (51.89197, -8.54919),
 (51.88993, -8.53861),
 (51.88892, -8.53033),
 (51.88887, -8.52555),
 (51.88896, -8.51953),
 (51.88845, -8.5149),
 (51.88879, -8.50994),
 (51.88896, -8.50781),
 (51.89009, -8.50677),
 (51.89205, -8.50618),
 (51.89364, -8.50033),
 (51.8942, -8.49751),
 (51.89472, -8.49465),
 (51.89541, -8.49066)]
group1 = lf.LayerGroup(layers=[lf.Marker(location=t) for t in stops[0:10]])
group2 = lf.LayerGroup(layers=[lf.Marker(location=t) for t in stops[10:20]])
themap.add_layer(group1)
themap.add_layer(group2)
themap.add_control(lf.LayersControl())
```

**Comment by martinRenou** [comment]
Thanks! I could reproduce it. I guess it's a bug, maybe we should give default layer names. I'm not entirely sure.

 When you create your groups you can give names:
```python
group1 = lf.LayerGroup(name='Group1', layers=[lf.Marker(location=t) for t in stops[0:10]])
group2 = lf.LayerGroup(name='Group2', layers=[lf.Marker(location=t) for t in stops[10:20]])
```

**Comment by undergroundquizscene** [comment]
Ah, that makes sense! Could this be explained in the documentation, either on [the LayerGroup page](https://ipyleaflet.readthedocs.io/en/latest/api_reference/layer_group.html) or on [the LayersControl page](https://ipyleaflet.readthedocs.io/en/latest/api_reference/layers_control.html)?

**Comment by martinRenou** [comment]
Sure, that would make sense. I leave this issue open until the documentation is fixed. Thanks for reporting! :)

**Comment by undergroundquizscene** [comment]
No problem, thanks very much!

**Comment by martinRenou** [comment]
Fixed by #319
